tensorbay/geometry/polygon.py

```diff
@@ -77,6 +77,12 @@ def __eq__(self, other: object) -> bool:
             return self._data.__eq__(other._data)
         return False
 
+    def __bool__(self) -> bool:
+        for point in self._data:
```

Comment:
bool for PointList2D should be the same as the bool of `list`
That means __bool__ return `False` if there are no point in the PointList, otherwise return `True`

Reply:
and `PointList2D` can use the `__bool__` from its superclass  "Sequence"
So there is no necessary to add `__bool__` for `PointList2D`, just test whether its `__bool__` behavior meets ourt expectation